src/datanode/src/region_server.rs

```diff
@@ -879,7 +879,8 @@ impl RegionServerInner {
             | RegionRequest::Alter(_)
             | RegionRequest::Flush(_)
             | RegionRequest::Compact(_)
-            | RegionRequest::Truncate(_) => RegionChange::None,
+            | RegionRequest::Truncate(_)
+            | RegionRequest::BulkInserts(_) => RegionChange::None,
```

Comment:
`Put` and `Delete` are categorized to `Ingest` type recently. We need to update against the latest main branch

Reply:
fixed in https://github.com/GreptimeTeam/greptimedb/commit/567a5413cbcffdc9fc2d4b13a1adddd042dab35a